content/source/assets/javascripts/analytics.js

```diff
@@ -13,4 +13,25 @@ document.addEventListener('turbolinks:load', function() {
       product: 'terraform'
     }
   })
+
+  track('.notification', function(el) {
```

Comment:
This is a super generic class - are we sure this isn't going to match anything else on the site? Increasing selector specificity here I think would help.

Reply:
The style is global and adds pretty specific look and feel. I could target only the instances that use an `<a>` since that sort of signals there's a next step to this particular `.notification` CTA

Reply:
Yeah anything we can get helps for now!